doc/meson.build

```diff
@@ -0,0 +1,128 @@
+configure_file(input: 'geany.1.in',
+               output: 'geany.1',
+               install: true,
+               install_dir: join_paths(join_paths(prefix, get_option('mandir'), 'man1')),
+               configuration: cdata)
+
+if rst2pdf.found()
+    custom_target('pdf-manual',
+                   input: ['geany.txt'],
+                   output: ['geany-@0@.pdf'.format(cdata.get('VERSION'))],
+                   command: [rst2pdf, '@INPUT@', '-o', '@OUTPUT@'],
+                   build_by_default: true)
+endif
+
+tarball = run_command('test', '-f', 'geany.html')
```

Comment:
Please specify `check: false`, because meson recently starts warning that the default is, unexpectedly, false and may change in future versions of meson.

Better yet, if your minimum version of meson allows it, use the `fs` module to check whether files exist without running external processes. :)

Reply:
I'm aiming to be compatible to Ubuntu 18.04, meaning meson 0.45. That has bitten a few times already ;-)

Reply:
Argh... :p 

It isn't the most recent LTS anymore, but well, if you need to support that still, I guess there is nothing to be done.

Reply:
@kugel- whilst its nice to support 18.04 as @eli-schwartz pointed out it is not longer the current LTS, but it is still in standard support until early next year.  But since this PR is a forward looking maybe we can skip the last year of 18.04 and move to 20.04 which has Meson 0.54 IIUC (unless you need 18.04 yourself of course :-).

Also, it may not be idiomatic, but in subdirectory `meson.build`s I always use `name = files("xxx")` to give a name to any file in the subdir that I need outside the subdir, that handles source/build and paths and all that stuff :-)

Reply:
Alright, convinced. 20.04 has 0.53 and that's the minimum required meson now.